CRM/Report/Form.php

```diff
@@ -1534,8 +1537,8 @@ public function addToDeveloperTab($sql) {
       foreach (['FROM', 'WHERE', 'GROUP BY', 'ORDER BY', 'LIMIT', ';'] as $term) {
         $sql = str_replace($term, '<br><br>' . $term, ($sql ?? ''));
       }
-      $this->sqlFormattedArray[] = $sql;
-      $this->assign('sql', implode(';<br><br><br><br>', $this->sqlFormattedArray));
+      $sqlFormattedArray = $sql;
```

Comment:
This doesn't look right. $sql is a string - it should be added to the array not replacing the array.

Reply:
@demeritcowboy yep - you are right - I wonder if that is what the tests are trying to tell me or not